.eslintrc.json

```diff
@@ -53,7 +53,9 @@
             "printWidth": 120,
             "semi": false
           }
-        ]
+        ],
+        "no-shadow": "off",
```

Comment:
in order to get typescript enums to work need to turn off `eslint` no shadow and use the typescript version instead